src/client.py

```diff
@@ -16,23 +17,22 @@ def client(message):
     message += '\r\n'
 
     if version_info[0] == 2:
```

Comment:
This isn't a bad approach (checking for version 2), but it might be more pythonic to use try/except.  At least you are not checking for version 3, which would be a bad idea, since python 4 would then break the program.